docs/api/rest_api/rest_api_reference/input/ez-cart.raml

```diff
@@ -4,22 +4,22 @@ post:
     description: Creates Cart from payload.
     headers:
         Content-Type:
-            description: The CartCreateStruct schema encoded in XML or JSON format.
+            description: The CartCreate schema encoded in XML or JSON format.
             example: |
-                application/vnd.ibexa.api.CartCreateStruct+xml
-                application/vnd.ibexa.api.CartCreateStruct+json
-    #body:
-    #    application/vnd.ibexa.api.CartCreateStruct+xml:
-    #        type: CartCreateStruct
-    #        example: !include examples/cart/POST/CartCreateStruct.json.example
-    #responses:
-    #    200:
-    #        body:
-    #            application/vnd.ibexa.api.Cart+json:
-    #                type: Cart
-    #                example: !include examples/cart/POST/Cart.json.example
-    #    400:
-    #        description: Error - The input does not match the input schema definition.
+                application/vnd.ibexa.api.CartCreate+xml
+                application/vnd.ibexa.api.CartCreate+json
+    body:
+        application/vnd.ibexa.api.CartCreate+json:
+            type: CartCreate
```

Comment:
This has to be added to ez-types.raml (I'm doing it.)

Reply:
Thanks, I forgot this part.

Reply:
@konradoboza Can you review my addition to ez-types.raml below?